lib/dunmanifestin/palette.rb

```diff
@@ -40,6 +55,16 @@ def file_basename
     File.basename(@filename, '.pal')
   end
 
+  def churn
+    @churn ||= Churn.new novelty: novelty do
+      phrases.sample.reify(genre, [])
```

Comment:
I like this! The `Churn` runs this block when called to generate, right? Somehow... I am looking at the code for `Churn#new` and I don't see where the block is consumed. Is this something implicit?

Reply:
Oh... no, this is a mistake. Or rather, a vestige of an earlier design, that turned out not to be workable.